api/views/plant.py

```diff
@@ -16,3 +18,10 @@ class PlantPartListView(ListAPIView):
     model = PlantPart
     queryset = PlantPart.objects.all()
     serializer_class = PlantPartSerializer
+
+
+class PlantCreateView(CreateAPIView):
+    model = Plant
+    queryset = Plant.objects.all()
+    serializer_class = PlantModificationSerializer
+    permission_classes = [IsAuthenticated, IsInstructor]
```

Comment:
```suggestion
    permission_classes = [IsInstructor]
```

Micro détail, `IsInstructor` vérifie déjà que l'utilisateur·ice soit authentifié : 

```python
class IsInstructor(permissions.BasePermission):
    def has_permission(self, request, view):
        return request.user.is_authenticated and InstructionRole.objects.filter(user=request.user).exists()
```